.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: ruby
 rvm:
-  - 1.9
-  - 2.0
-  - 2.1
```

Comment:
This mean we don't support Ruby < 2.4. We should make this explicit in the gemspec.

Reply:
I changed it to test additional versions instead.